src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -132,3 +132,65 @@
     line-height: 1.3;
   }
 }
+
+.barStat {
+  display: flex;
+}
+
+.barStat.vertical {
+  flex-direction: row;
+  gap: 26px;
+}
+
+.barStat.horizontal {
+  flex-direction: column;
+  gap: 24px;
+}
```

Comment:
This is really clever and slick!

However, I'm seeing an issue with how the bars are aligned in the horizontal variant:
<img width="348" alt="Screen Shot 2022-03-02 at 9 43 17 PM" src="https://user-images.githubusercontent.com/1002748/156503737-83cf5311-6511-4b4e-ae76-7ac47a0c3e5d.png">

It's going to be tricky to align the bars with your two-level flexbox approach here, since the widths of the labels is not known statically. You _might_ be able to pull this off with CSS grid, which should allow the width of the widest label determine the width of all other labels in the column, but it'll require restructuring your HTML and CSS a bit. On the plus side, if we stop precisely computing the lengths of the bars and just stick with some fixed lengths, I think it'll simplify this by a lot.

If you try out the grid approach, one piece of advice I have is to treat the bar and the statistic number as a single column/row. This is because the numbers are _not_ aligned with each other, and they should appear right after the bar. In other words, you'll want a 3x2 (or 2x3) grid, not a 3x3 grid, where the label is one row/column and the bar + number combined is the other row/column.